lab-trent/test/api-test.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
```

Comment:
Yeah, would like to see some additional tests, but overall really solid work on the implementation.